readthedocs/doc_builder/environments.py

```diff
@@ -295,8 +313,9 @@ def run(self):
             # is in the last 15 lines of the command's output
             killed_in_output = 'Killed' in '\n'.join(self.output.splitlines()[-15:])
             if self.exit_code == DOCKER_OOM_EXIT_CODE or (self.exit_code == 1 and killed_in_output):
-                self.output = _('Command killed due to excessive memory '
-                                'consumption\n')
+                self.output += str(_(
+                    'Command killed due to excessive memory consumption\n'
```

Comment:
I would pre-prend `\n\n` so it's visually easy to see the last line.

Reply:
I was thinking something like that, isn't just one enough, tho?

Reply:
I think it's better to have a blank line that separates what's real output and what's what we added.